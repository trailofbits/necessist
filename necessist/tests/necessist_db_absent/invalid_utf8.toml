args = ["--no-sqlite", "--root=examples/invalid_utf8"]
stderr = ""

[bin]
name = "necessist"

[fs]
cwd = "../../.."

[status]
code = 0
