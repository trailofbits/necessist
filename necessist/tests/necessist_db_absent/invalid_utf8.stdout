3 candidates in 1 test file
examples/invalid_utf8/src/lib.rs: dry running
examples/invalid_utf8/src/lib.rs: mutilating
examples/invalid_utf8/src/lib.rs:8:5-8:19: Warning: invalid utf-8 sequence of 1 bytes from index 0: "/xFE"`
Silence this warning with: --allow output-invalid
examples/invalid_utf8/src/lib.rs:8:5-8:19: `write = false;` passed
examples/invalid_utf8/src/lib.rs:11:9-11:49: `stdout().write(&[0xfe, b'/n']).unwrap();` passed
